### PR TITLE
Imagebasemaps fixes

### PR DESCRIPTION
**Pull request by sdamaraju**

### CODE REVIEW

**Comment by sdamaraju** [comment]
Fixes issue #317 and  #306

**Comment by NathanNovak** [comment]
@sadamaju The pano images are sized correctly but are still showing up as a black rectangle.Here is the line :
Changed image basemap to: {id: 16197086516866, height: 1071, width: 4776, annotated: true, title: 16197086516866}

File: src/modules/images/useImages.js [hunk 1]
```diff
@@ -150,12 +151,26 @@ const useImages = () => {
     return Platform.OS === 'ios' ? imageURI : 'file://' + imageURI;
   };
 
+  const resizeImageIfNecessary = async (imageData) => {
+    let height = imageData.height;
+    let width = imageData.width;
+    const tempImageURI = Platform.OS === 'ios' ? imageData.uri || imageData.path : imageData.uri || 'file://' + imageData.path;
+    if (!height || !width) ({height, width} = await getImageHeightAndWidth(tempImageURI));
+    let resizedImage = {};
+    if (height > (deviceDimensions.width * 4) || width > (deviceDimensions.width * 4)) {
```
Comment (NathanNovak) [inline src/modules/images/useImages.js:160]: deviceDimensions.width is here twice.  The first arg should be deviceDimensions.height

**Comment by NathanNovak** [comment]
Ok great that works now.

**Comment by NathanNovak** [comment]
Images are in the correct orientation when importing them and viewing them as a image basemap too.

**Pull request merged**